### PR TITLE
Solved some issues

### PR DESCRIPTION
**Pull request by SoanKim**
Updated
1. lunar_lander.ipynb (solved: pip install gym[box2d] error)
2. cellular_segmentation.ipynb (solved: numpy long type problem. unsolved: "https://www.suite2p.org/test_data/gt1.tif" link doesn't exist)
3. blurry_vision.ipynb (solved: cats-and-dogs.zip not found)

### CODE REVIEW

File: projects/ReinforcementLearning/lunar_lander.ipynb [hunk 1]
```diff
@@ -771,36 +778,176 @@
...
+    "            # which is proven to never be periodic, k = 0.01\n",
+    "            wind_mag = (\n",
+    "                math.tanh(\n",
+    "                    math.sin(0.02 * self.wind_idx)\n",
+    "                    + (math.sin(math.pi * 0.01 * self.wind_idx))\n",
+    "                )\n",
+    "                * self.wind_power\n",
+    "            )\n",
+    "            self.wind_idx += 1\n",
+    "            self.lander.ApplyForceToCenter(\n",
```
Comment (iamzoltan) [inline projects/ReinforcementLearning/lunar_lander.ipynb:801]: did you implement this for them?

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Looking pretty good! Great work. Some minor tweaks and comments to address

**Comment by iamzoltan** [comment]
It also looks like your branch have conflicts that need to be resolved. Please address them

File: projects/ReinforcementLearning/lunar_lander.ipynb (hunk 1 above)
Comment (SoanKim) [reply projects/ReinforcementLearning/lunar_lander.ipynb]: No, the lines 793-801 are the original ones.

File: projects/ReinforcementLearning/lunar_lander.ipynb [hunk 2]
```diff
@@ -771,36 +778,176 @@
...
+    "        if self.game_over or abs(state[0]) >= 1.0:\n",
+    "            terminated = True\n",
+    "            reward = -100\n",
+    "        if not self.lander.awake:\n",
+    "            terminated = True\n",
+    "            reward = +100\n",
+    "\n",
+    "        if self.render_mode == \"human\":\n",
+    "            self.render()\n",
+    "        return np.array(state, dtype=np.float32), reward, terminated, False, {}"
```
[the remark this answers is not in the document]
Comment (SoanKim) [reply projects/ReinforcementLearning/lunar_lander.ipynb]: I'm sorry, but I don't see it.

File: projects/ReinforcementLearning/lunar_lander.ipynb [hunk 3]
```diff
@@ -100,11 +102,15 @@
...
     "!pip install ale-py --quiet\n",
+    "!pip install -q swig --quiet\n",
     "!pip install gym[box2d] --quiet\n",
     "!pip install pyvirtualdisplay --quiet\n",
     "!pip install pyglet --quiet\n",
     "!pip install pygame --quiet\n",
-    "!pip install minigrid --quiet"
+    "!pip install minigrid --quiet\n",
+    "!pip install -q gymnasium[box2d] --quiet\n",
+    "!pip install 'minigrid<=2.1.1' --quiet\n",
```
[the remark this answers is not in the document]
Comment (SoanKim) [reply projects/ReinforcementLearning/lunar_lander.ipynb]: I've deleted the former one.

**Review by iamzoltan — CHANGES_REQUESTED** [review]
@SoanKim on line 954
`"## Enter the name of the custome environment you created and uncomment the line below.\n`

Can you address this small fix?

File: projects/ReinforcementLearning/lunar_lander.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (iamzoltan) [reply projects/ReinforcementLearning/lunar_lander.ipynb]: line 954
`"## Enter the name of the custome environment you created and uncomment the line below.\n`

**Comment by iamzoltan** [comment]
Its all good, ill sort it out.

**Pull request merged**